### PR TITLE
Feature/l2 get request systemuser

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Description

In Controller only ( service layer mocks the response)

GET by Request.Id Guid
GET by ExternalRef + SystemId

## Related Issue(s)
- #586

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

### CODE REVIEW

File: src/Authentication/Services/RequestSystemUserService.cs [hunk 1]
```diff
@@ -13,6 +13,11 @@ public async Task<CreateRequestSystemUserResponse> CreateRequest(CreateRequestSy
     {
         await Task.Delay(65);
```
Comment (TheTechArch) [inline src/Authentication/Services/RequestSystemUserService.cs:14]: Hva er grunnen til denne?

File: src/Authentication/Services/RequestSystemUserService.cs [hunk 2]
```diff
@@ -25,4 +30,40 @@ public async Task<CreateRequestSystemUserResponse> CreateRequest(CreateRequestSy
             SystemUserId = Guid.NewGuid().ToString()
         };
     }
+
+    /// <inheritdoc/>
+    public async Task<CreateRequestSystemUserResponse> GetRequestByExternalRef(string systemId, string externalRef1)
+    {
+        await Task.Delay(65);
```
Comment (TheTechArch) [inline src/Authentication/Services/RequestSystemUserService.cs:37]: Hvorfor delay?

Comment (simen-rekkedal) [reply src/Authentication/Services/RequestSystemUserService.cs:37]: Bare så lenge service laget er mocked, fordi jeg er alergisk mot warnings om at det mangler await i en async metode. Ingen egentlig funksjon. :-)

File: src/Authentication/Services/RequestSystemUserService.cs (hunk 1 above)
Comment (simen-rekkedal) [reply src/Authentication/Services/RequestSystemUserService.cs:14]: fjernes når jeg dev'er service laget

**Pull request merged**